pkg/kf/commands/routes/unmap_route.go

```diff
@@ -102,8 +102,10 @@ func NewUnmapRouteCommand(
 				return fmt.Errorf("failed to unmap Route: %s", err)
 			}
 
+			app, _ := appsClient.Get(cmd.Context(), p.Space, appName)
```

Comment:
I _think_ appsClient.Transform returns the updated application so you could use that instead.

Reply:
It does! Great, switching over to that.